css/css-grid/layout-algorithm/grid-as-flex-item-should-not-shrink-to-fit-005.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Evaluate the behavior of a grid container as Flexbox item</title>
+<link rel="author" title="Javier Fernandez Garcia-Boente" href="mailto:jfernandez@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#layout-algorithm">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#flex-lines">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#propdef-flex-wrap">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#propdef-align-content">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#propdef-align-self">
+<link rel="help" href="https://drafts.csswg.org/css-align-3/#align-flex">
+<link rel="help" href="https://drafts.csswg.org/css-align-3/#align-grid">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht">
+<meta name="assert" content="A single-line column flexbox shouldn't shrink-to-fit since its flex item is stretching in the main-axis, honoring the 'align-self: normal (behaves as 'stretch')'. The grid item's inline-size should be stretched as well, honoring its justify-self: 'normal' (behaves as 'stretch')">
+<link rel="stylesheet" href="/fonts/ahem.css">
+<style>
+    body { overflow: hidden; }
+    .flexbox {
+        display: flex;
+        flex-flow: column nowrap;
+        align-content: flex-start;
+        width: 100px;
+        height: 100px;
+        font: 25px/1 Ahem;
+        color: green;
+    }
+    .grid {
+        display: grid;
+        align-items: start;
```

Comment:
So these last two test baseline vs start? Did that difference matter for webkit? Otherwise this seems not super useful, though I'm also fine with including that.

Reply:
Well, these tests are kind of regression tests for a bug we detected in webkit, caused by the use of align-items: baseline in combination with other properties. The test cases without baseline are kind of control cases; any value different than baseline would be valid for this purpose.

Reply:
Sounds good, thanks.